procurement_portal/records/apps.py

```diff
@@ -1,8 +1,21 @@
 from django.apps import AppConfig
+from django.test import Client
```

Comment:
let's not use the test client in prod. Are you looking for an http client? we usually use requests.